cmd/legacy.go

```diff
@@ -65,6 +65,7 @@ func init() {
 	legacyCmd.Flags().StringVar(&monitorConfig.FromBlock, "from-block", "", "Begin indexing the chain from this block. Can pass a number or \"latest\"")
 	legacyCmd.Flags().IntVar(&monitorConfig.Concurrency, "concurrency", 10, "Process concurrency, how many go routines to fire in parallel")
 	legacyCmd.Flags().StringVar(&cpuprofile, "cpuprofile", "", "write cpu profile to file")
+	legacyCmd.Flags().DurationVar(&monitorConfig.TraceTransactionTimeout, "traceTransactionTimeout", time.Second*50, "The timeout used for debug_TraceTransaction calls. The default is 50s which was previously hardcoded, a safe value to trace all transactions as of Jan 2022 would be 120s")
```

Comment:
Preference: I'd be in favour of not adding the comment to the CLI help.
```suggestion
	legacyCmd.Flags().DurationVar(&monitorConfig.TraceTransactionTimeout, "traceTransactionTimeout", time.Second*120, "The timeout used for debug_TraceTransaction calls.")
```